docs/SBVR.md

```diff
@@ -0,0 +1,355 @@
+# SBVR in Pine.js
+
+An SBVR model is composed of one or more _vocabularies_, serving as namespaces. Each vocabulary is composed of _terms_, _fact types_, and _rules_, which together define the data model and its constraints. Terms from different vocabularies can be referenced adding the name of the vocabulary, in parenthesis, after the term.
+
+SBVR files are composed of statements, one per line, in the form `<header>: <body>`. Indentation is not significant, and `--` introduces comments that go until the end of the line. SBVR files accepted by Pine.js are case-insensitive and usually have the `.sbvr` extension. `Vocabulary`, `Term`, `Fact Type`, and `Rule` are toplevel declarations that can be followed by _attributes_. See the **Attributes** subsection below.
+
+A vocabulary can be started with:
+
+```
+Vocabulary: <vocabulary-name>
+```
+
+Every new definition is part of the last declared vocabulary. If no vocabulary has been explicitly declared, a `Default` vocabulary is implicitly assumed. Within vocabularies, term declarations introduces _types_:
+
+```
+Term: <term-name>
+```
+
+where `<term-name>` is the name of the term. Because SBVR focuses on structuring natural languages, terms can have spaces in their names. By default, every instance of a term is identified by an `id` and persisted into a table.
```

Comment:
Maybe we can phrase this differently, as Concept-types (introduced later) make this technically not true (and its not just an implementation detail, most of the terms you define are in fact columns). The intuition to convey is that terms are the building blocks of our objects, and terms themselves can be formed by composing other terms. In the context of a relational DB it is easy to see how this maps to tables and columns.